server/cmwell-ws/app/security/httpauth/BasicHttpAuthentication.scala

```diff
@@ -23,15 +23,12 @@ import scala.util.{Failure, Success, Try}
 trait BasicHttpAuthentication {
   private val authTypePrefix = "Basic "
 
-  def decodeBasicAuth(auth: String): Try[(String, String)] = {
-    Try(new String(new Base64().decode(auth.drop(authTypePrefix.length)), "UTF-8")) match {
-      case Success(userColonPass) =>
-        val split = userColonPass.split(":", 2)
-        if (split.length != 2)
-          Failure(new IllegalArgumentException(s"$auth is not a valid Basic header"))
-        else
-          Success(split(0)->split(1))
-      case Failure(_) => Failure(new IllegalArgumentException(s"$auth is not a valid Basic header"))
-    }
+  def decodeBasicAuth(auth: String): (String, String) = {
+    val userColonPass = new String(new Base64().decode(auth.drop(authTypePrefix.length)), "UTF-8")
```

Comment:
Use `cmwell.util.string.Base64`, and it might be worth validating before decoding, with: `Base64.isBase64(...)`.
No need for `new` instance